app/v2/notifications/post_notifications.py

```diff
@@ -135,6 +140,24 @@ def post_precompiled_letter_notification():
     return jsonify(resp), 201
 
 
+def _seed_bounce_data(epoch_timestamp: int, service_id: str):
```

Comment:
Just want to make sure I understand the use cases for this function:
1. When the feature is first launched we would set `epoch_timestamp = current_time - 24_hours`?
2. If we lose all our redis data we would also set this to 24 hours to recover?
3. If there is some kind of outage and we haven't been able to write to redis for, say 5 hours, we would set `epoch_timestamp = current_time - 5_hours`?

Reply:
Case 3 is not relevant - we will just use this if we lose all data from Redis, and write the last 24 hours.